docs/core/development/policy/coding-standard.rst

```diff
@@ -76,18 +76,19 @@ Indentation is 4 spaces per indent.
 Tabs are not allowed.
 It is preferred that every block appears on a new line, so that control structure indentation is always visible.
 
-Lines are flowed at 79 columns.
```

Comment:
1. Can we just remove every aspect of this document covered by Black and just say "code must be formatted by Black"?
2. Given that this is one of the very few options Black supports, I'd really prefer to keep the 79-column line length limitation.  For one thing, it matches the docstring width, and for another, 79 columns is of a size where someone with average eyesight can comfortably fit 2 columns on a 15" laptop screen; 89 is too wide to practically make use of that space.  (If it weren't configurable I don't feel strongly enough about this to want to patch Black or anything, but it seems to be aware that 89 is a peculiar choice.)

Reply:
(The 79-column thing is not a blocker, if this is something you disagree with I'm fine being overridden here.)